### PR TITLE
Modified mysql-backup spec file

### PR DESCRIPTION
**Pull request by snigdha-px**
**What this PR does / why we need it**:
Removed group and secret parameter
Deleted snapshot file from mysql-backup which is not required
**Which issue(s) this PR fixes** (optional)
Closes #https://portworx.atlassian.net/browse/PA-1260

**Special notes for your reviewer**:

https://aetos.pwx.purestorage.com/resultSet/testSetID/254078

### CODE REVIEW

**Review by kphalgun-px — APPROVED** [review]
LGTM

File: drivers/scheduler/k8s/specs/mysql-backup/px-mysql-storage.yaml [hunk 1]
```diff
@@ -3,18 +3,12 @@ kind: Secret
...
-data:
-  mysql-secret: WW91IHNuZWFreSBsaXR0bGUgcGlnbGV0IQ==
 ---
 ##### Portworx persistent volume claim
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: mysql-data
-  annotations:
-    px/secret-name: volume-secrets
```
Comment (vpinisetti-px) [inline drivers/scheduler/k8s/specs/mysql-backup/px-mysql-storage.yaml:15]: When `volume-secrets` are not being used, better to remove entire secret object itself.

**Review by vpinisetti-px — CHANGES_REQUESTED** [review]

File: drivers/scheduler/k8s/specs/mysql-backup/px-mysql-storage.yaml (hunk 1 above)
Comment (snigdha-px) [reply drivers/scheduler/k8s/specs/mysql-backup/px-mysql-storage.yaml:15]: done

**Review by vpinisetti-px — APPROVED** [review]
LGTM

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Review by ak-px — APPROVED** [review]
LGTM

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Pull request merged**